.github/workflows/run-litmus.yml

```diff
@@ -1,6 +1,10 @@
 name: Run Litmus
 
 on:
+  pull_request:
+    paths:
+      - '.github/workflows/run-litmus.yml'
+      - 'litmus/**'
```

Comment:
Does this mean that we only run when there are changes in `litmus/**'? If yes should we also add `lib/`? Or would that create a lot of false positives (changes in lib that won't have an impact on litmus).

Reply:
@maranget was not a fan, and probably meaningful changes in `lib` would have an impact on files in `litmus` anyway so maybe it's fine? Happy to be told otherwise